python/cudf/cudf/groupby/groupby.py

```diff
@@ -52,6 +53,22 @@ def size(self):
         data = cudf.Series(column_empty(nrows, "int8", masked=False))
         return data.groupby(self._groupby.key_columns).count()
 
+    def serialize(self):
+        header, frames = self._groupby.serialize()
+        header["type"] = pickle.dumps(type(self))
+        return header, frames
```

Comment:
So we can ignore serializing the dataframe / series here because it will be serialized by the `_groupbyhelper` object and we can use that to reconstruct the `_groupby` object?

Reply:
Yes, precisely